CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 - Update to Composer 2.4 (#513)
 - Change the default indentation for rst files to 4 spaces (#194)
+- Use CamelCase for the TsConfig folder (#522)
```

Comment:
Newest changes on top, please (as the changelog is in reverse chronological order).

Reply:
Done